kernel/src/main/java/org/kframework/unparser/OutputModes.java

```diff
@@ -7,5 +7,18 @@
  *
  */
 public enum OutputModes {
-    PRETTY, PROGRAM, KAST, BINARY, NONE
+    PRETTY, PROGRAM, KAST, BINARY, NONE;
+
+    private String extension;
```

Comment:
this seems to be unused?

Reply:
This change facilitates a future PR, I'll move the commit to that PR instead.